tests/shadow/test_shadow_transforms.py

```diff
@@ -308,22 +331,6 @@ def test_backward_torch(self):
 class TestExpvalTransform:
     """Test that the expval transform is applied correctly"""
 
-    def test_hadamard_transform(self):
```

Comment:
Why was this test removed?

Reply:
This test only checks that the `_replace_obs` transform works correctly. I added these tests above.